src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+    protected String items;
+    protected Boolean isChecked;
+    protected String tag;
+
+    public Task(String items){
```

Comment:
Some of the functions have a white space before parentheses, some does not. Should it be more consistent?